theorems/T000488.md

```diff
@@ -2,13 +2,13 @@
 uid: T000488
 if:
   and:
+    - P000154: true
     - P000019: true
-    - P000133: true
 then:
   P000020: true
 refs:
   - mathse: 1706258
     name: Every sequence has a monotone subsequence
 ---
 
-Every sequence in a totally ordered set has a monotone subsequence (see for example {{mathse:1706258}}). The space being {P19} implies that this subsequence has an accumulation point, which must be its limit since the subsequence is monotone.
+Every sequence in a totally ordered set has a monotone subsequence (see for example {{mathse:1706258}}). The space being {P19} implies that this subsequence has an accumulation point, and every order-convex open neighborhood of the accumulation point contains eventually the subsequence as it contains infinitely many terms.
```

Comment:
Maybe slightly more for the last sentence:

```suggestion
Every sequence in a totally ordered set has a monotone subsequence (see for example {{mathse:1706258}}). The space being {P19} implies that this subsequence has an accumulation point $x$.  And because the subsequence is monotone, it converges to $x$ because every order-convex neighborhood of $x$ contains eventually the subsequence as it contains infinitely many terms of it.
```